### DISCUSSION TITLE
Some questions about 'Defining Cluster Identity with scRNA-seq'

### DISCUSSION THREAD

**Discussion by Rui-Jing** [GreenleafLab/ArchR#1494] [Questions / Documentation] [answered]
Hi, Thanks for developing this user-friendly tool.  I have some questions about defining cell identity in ScATAC-seq.

1. Whether cell identities can be defined without scRNA-seq data?  In chapter 7 of  ArchR munual,  we can get a series of marker genes. Can I directly define cell identity based on these predicted genes without corresponding RNA sequencing samples? If so, how to do it in detail?

2.  Some cells are not detected by scRNA, but these cells are detected by scATAC, so How do I define their cellular identity without deleting them? In the  constrained integration, can these clusters be defined manually first and then other groups can be defined by scRNA-seq? 

3. In the constrained integration, A cluster of ScATAC-seq is defined as four subgroups of neurons, which fits my biological explanation. But when I label the cell identity, the **algorithm's **optimal annotation**** of the cell  is not what I want. How do I mark a Cluster with four identities. (Does ArchR have a function like Seurat's **FindSubCluster** to find subgroups)

the algorithm's optimal annotation is **labelNew**
```
cM <- confusionMatrix(projHeme3$Clusters, projHeme3$predictedGroup)
labelOld <- rownames(cM)
labelOld

labelNew <- colnames(cM)[apply(cM, 1, which.max)]
labelNew
```

I feel very sorry for many questions.

**Comment by rcorces** [comment, marked as answer]
The chapters on constrained and unconstrained integration are by no means required. You can annotate cluster identity however you see fit. If you only had ATAC-seq data, you would likely do this using gene activity scores for marker genes.

We do not have a sub clustering function. You can manually do subclustering by subsetting your project using the `subsetArchRProject()` function

**Comment by Rui-Jing** [comment]
Thanks for your quickly reply. I'm still a little confused. 

1. If I only have scATAC-seq data, Is it sufficient to label cell identity with the following code?(Of course, I first identified the cell identities according to the marker genes)
`projHeme3$Clusters2 <- mapLabels(projHeme3$Clusters, newLabels = labelNew2, oldLabels = labelOld)`

2. Another question is manual annotation with constrained integration when annotating scATAC data. First, **Selecting a portion of the cells for manual labeling, then constrained integration of the remaining cells, and finally merged them together.** I don't know whether such an idea can be realized. Could you give me some help?

**Reply by rcorces** [reply]
1. sure
2. Seems reasonable. You can annotated whatever information you want in a new column of cellColData. So you can label your cells whatever you want based on your knowledge of your biological system

**Reply by Rui-Jing** [reply]
Great. Thanks for your kind help.
